src/pages/projects.jsx

```diff
@@ -59,6 +60,48 @@ const ProjectImage = styled.img`
     max-width: 100%;
 `;
 
+const Player1 = ({ classCircles }) => (

```

Comment:
Give this a more specific name. Instead of `Player1`, you can call this `CirclesVideo`. (The styled component you currently have as `CirclesVideo` should be renamed based on  [this comment](https://github.com/dollarbeale/website_project/pull/41/files#r1127182749))